.github/labeler.yml

```diff
@@ -0,0 +1,8 @@
+critical:
```

Comment:
Do I need to create something to match those in the Project, or by selecting `Bug Triage` it is done by default?

Reply:
Those are for labels

Reply:
Issue labeler